### DISCUSSION TITLE
Rendering Markush structures in Indigo

### DISCUSSION THREAD

**Discussion by shixuanleong** [epam/Indigo#2092] [Q&A] [answered]
Hello, I would like to construct different Markush structures with frequency and positional variations (Examples below). Are there any options for doing this? 

![image1](https://github.com/epam/Indigo/assets/84304673/7ef9e3b5-bb5f-4eb0-947b-aac3f8ad04c3)

**Comment by AlexanderSavelyev** [comment, marked as answer]
Hi @shixuanleong . Unfortunately markush structures rendering is not supported in Indigo
